ipfx/bin/generate_pipeline_input.py

```diff
@@ -68,50 +68,63 @@ def get_input_h5_file_from_lims(specimen_id):
     return h5_file_name
 
 
-def make_input_json(args, plot_figures=False):
```

Comment:
I wonder if it is Ok to pass arguments this way when args here are actually keyword arguments